tests/components/test/test_config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Test the testing config flow."""
```

Comment:
This file doesn't belong with this integration?

Reply:
removed